docs/installation.md

```diff
@@ -1,3 +1,13 @@
+# Quick install
+
+Run below command to install latest version of aeternity node.
+
+```bash
+wget https://raw.githubusercontent.com/aeternity/epoch/master/scripts/install.sh && chmod +x install.sh && ./install.sh 1.1.0
```

Comment:
I am a bit concerned of this line containing `master` in a doc meant to be tagged. Is this rather meant to be reformulated as a section in the /README.md pointing to a wiki page?

Reply:
Well, maybe it's time to start maintaining `latest` (or `stable` or whatever name)  branch that links to latest release so it can be used in the documentation/tutorials.

Reply:
I don't want to hide this "on-liner" in the wiki. Not sure about master reference tho, maybe it's better to reference on tag, until we have "latest" ?

I don't like having commands even in the master docs that don't actually work, like `install 1.2.0` ..

Reply:
> I don't like having commands even in the master docs that don't actually work (yet), like `install 1.2.0` ..

This means there is a small risk that these required changes may go unnoticed before tagging. It would be great if you added specific notes in https://github.com/aeternity/epoch/wiki/Release-process